source/test/TestApp/PlayFabEventTest.cpp

```diff
@@ -218,6 +229,40 @@ namespace PlayFabUnit
         std::bind(&PlayFabEventTest::NonStaticEmitEventCallback, this, std::placeholders::_1, std::placeholders::_2));
     }
 
+    void PlayFabEventTest::GenericMultiThreadedTest(unsigned int pNumThreads, unsigned int pNumEventsPerThread)
```

Comment:
>p [](start = 65, length = 1)

do you guys use hungarian? what does this 'p' prefix mean?

Reply:
parameter, this was meant to be distinctly different from the previous static name, so it's now just excessive. 

But as it's still a parameter, it still makes some sense (but admittedly it's not consistent throughout)

---
In reply to: [345443034](https://github.com/PlayFab/XPlatCoreTemplate/pull/44#discussion_r345443034) [](ancestors = 345443034)

Reply:
I don't think it makes sense if this style isn't consistent with any other method. it's just visual noise (and I don't think parameter is a typical prefix that is even used by those who use hungarian notation so it's double confusing. reactivating for consideration).

---
In reply to: [345516951](https://github.com/PlayFab/XPlatCoreTemplate/pull/44#discussion_r345516951) [](ancestors = 345516951,345443034)

Reply:
actually I did some more googling and I see some people doing  this in other places but not calling it hungarian (it's more like the 'm' prefix). that's fine i'll bite on this.

---
In reply to: [345524976](https://github.com/PlayFab/XPlatCoreTemplate/pull/44#discussion_r345524976) [](ancestors = 345524976,345516951,345443034)